src/views/Auth/Auth.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { useState } from 'react';
+import './Auth.css';
+import { signInUser, signUpUser } from '../../services/users';
+
+export default function Auth({ setCurrentUser }) {
+  const [isSignUp, setIsSignUp] = useState(true);
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState('');
+
+  const toggleAuth = () => {
+    setIsSignUp(!isSignUp);
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    if (isSignUp) {
+      try {
+        const resp = await signUpUser(email, password);
+        setCurrentUser(resp.email);
+      } catch (e) {
+        setError(e.message);
+      }
+    } else {
+      try {
+        const resp = await signInUser(email, password);
+        setCurrentUser(resp.email);
+      } catch (e) {
+        setError(e.message);
+      }
+    }
+  };
+
+  return (
```

Comment:
Great work in this file updating state passed on what  the user is wanting to do.